src/World.cpp

```diff
@@ -146,7 +150,7 @@ cWorld::cWorld(const AString & a_WorldName, const AString & a_DataPath, eDimensi
 	m_LastChunkCheck(0),
 	m_LastSave(0),
 	m_SkyDarkness(0),
-	m_GameMode(gmNotSet),
+	m_GameMode(gmSurvival),
```

Comment:
This is an unrelated change, should not be in this PR

Reply:
It was set in start before, now it is set in the constructor.  This is entirely related.

Reply:
Okay, didn't see that.